src/client/css/partials/breachDetails.css

```diff
@@ -6,6 +6,21 @@
   padding: var(--padding-xl) var(--padding-sm);
 }
 
+.breach-detail-header .breach-logo {
+  height: 5rem;
+  display: inline-block;
+}
+
+.breach-detail-header span.breach-logo {
+  line-height: 5rem;
+  width: 5rem;
+  border-radius: 5rem;
+  font-size: 3rem;
+  font-weight: bold;
+  text-transform: uppercase;
```

Comment:
Does `text-transform: uppercase;` have localization implications?  I recall some research @flodolo might have been part of – maybe this just works 🤞, or maybe we need to use `toLocaleUpperCase()`?

Reply:
Don't…
https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#css-issues

Reply:
One thing that I don't understand from this patch is what this text is? Is it the name of the breached company, we shouldn't make decision on how their name is capitalized.